### ISSUE TITLE
Optional Arguments?

### ISSUE THREAD

**Issue by calebthebrewer** [calebmer/postgraphql#184]
I ran into an issue passing arguments to `postgraphql(...)`. I expected the `schemaName` to be optional after reading [the documentation](https://github.com/calebmer/postgraphql/blob/master/docs/library.md#postgraphqlpgconfig-schemaname--public-options), so I wrote the following:
```js
postgraphql('postgres://localhost:5432', { graphiql: true })
```
No errors are thrown and the `options` argument is used as the `schemaName`, which means `graphiql` isn't running and PostGraphQL doesn't successfully connect. This isn't really a bug, but it may be helpful to point out that, if you want to specify `options`, you _must_ specify a `schemaName`:
```js
postgraphql('postgres://localhost:5432', 'public', { graphiql: true })
```
Additionally, it would be nice if the `schemaName` wasn't required in order to pass `options`, but that's just a nice-to-have.

(Currently using v2.1.1)

**Comment by calebmer** [comment]
Thanks so much for reporting, yeah this is a bug. I’ll look into this 👍

**Comment by calebmer** [comment]
Fixed in `2.2.1` 💫
